monitoring/kube-prometheus-stack/prometheus-remote-write-aws.yaml

```diff
@@ -1,14 +0,0 @@
-prometheus:
```

Comment:
Shall we keep this file? It's used as a starting example and referenced in the README.